packages/kernel-6.12/kernel-6.12.spec

```diff
@@ -314,7 +289,6 @@ install -d %{buildroot}%{_cross_libexecdir}/neuron/neuron_latest/
 mv %{buildroot}%{_cross_kmoddir}/neuron_2_21/neuron.%{_ko} %{buildroot}%{_cross_libexecdir}/neuron/neuron_2_21/
 mv %{buildroot}%{_cross_kmoddir}/neuron_latest/neuron.%{_ko} %{buildroot}%{_cross_libexecdir}/neuron/neuron_latest/
 %endif
-mv %{_builddir}/efa_driver/build/src/efa.%{_ko} %{buildroot}%{_cross_kmoddir}/kernel/drivers/amazon/net/efa/
```

Comment:
Can we exclude the in-tree EFA since we don't plan to ever use it, its just using up space on the image.